site/app/templates/forum/ThreadPostForm.twig

```diff
@@ -164,29 +164,34 @@
     {% endif %}
     <fieldset class="inline-block right-zero">
       <legend style="display:none">Settings:</legend>
+        <div class="new-thread-settings-wrapper">
```

Comment:
You can add the style `align-items: center;` to this class to address William's comment about the vertical alignment